metrics/grafana/tikv_details.json

```diff
@@ -16856,11 +16966,18 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(delta(tikv_raftstore_snapshot_total{tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m])) by (type, status)",
+              "expr": "sum(rate(tikv_raftstore_snapshot_total{tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m])) by (type, status)",
```

Comment:
Why change this? Actually I think `delta` is more straightforward here. For example, 2 snapshot in 60 seconds can be 0.03 with rate, but can show 2 if using delta.

Reply:
OK.... I used to think it is not accurately.